### PR TITLE
feat: cart page complete; import: style;

### PR DESCRIPTION
**Pull request by Gautam2305**
- cart page with order details
- price , discounted price and total price also shown
- delivery charges also shown
- two primary CTAs to remove from cart and to move the item to wishlist
- one secondary CTA to place order 

Screenshot (124)

### CODE REVIEW

File: pages/cart_page/cart.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.cart-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (ttahm3d) [inline pages/cart_page/cart.css:3]: If you wanted to achieve responsiveness, add a media query. This is will look too small on mobile screens

File: pages/cart_page/cart.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    max-width: 60%;
+}
+.prod-container{
+    display: grid;
+    grid-template-columns: repeat(2,200px);
+    align-items: center;
+    justify-items: center;
+    max-width: 40%;
```
Comment (ttahm3d) [inline pages/cart_page/cart.css:11]: Giving max width 40% defats the entire purpose of grid.

**Review by ttahm3d — APPROVED** [review]
It looks good.have added few comments which can be improved on later

File: pages/cart_page/cart.css (hunk 1 above)
Comment (Gautam2305) [reply pages/cart_page/cart.css:3]: it is not yet made for mobile screens...i will make it responsive later on while refactoring

File: pages/cart_page/cart.css (hunk 2 above)
Comment (Gautam2305) [reply pages/cart_page/cart.css:11]: done...thanks for the review

**Pull request merged**